project/Dependencies.scala

```diff
@@ -21,65 +21,61 @@ object Dependencies {
   // by being listed here.
   // One reason to specify an override here is to avoid static-analysis security warnings.
   val transitiveDependencyOverrides: Seq[ModuleID] = Seq(
-    "com.google.guava"           % "guava"                      % "33.4.8-jre",
-    "com.fasterxml.jackson.core" % "jackson-annotations"        % jacksonV,
-    "com.fasterxml.jackson.core" % "jackson-databind"           % jacksonHotfixV,
-    "com.fasterxml.jackson.core" % "jackson-core"               % jacksonV,
-    "org.yaml"                   % "snakeyaml"                  % "2.4",
-    "org.apache.commons"         % "commons-compress"           % "1.27.1", // workbench-libs libraries pull this in
-    "com.google.apis"            % "google-api-services-pubsub" % "v1-rev20250414-2.0.0", // from workbench-google2
-    "com.google.apis"  % "google-api-services-admin-directory"  % "directory_v1-rev20250626-2.0.0" // from workbench-google2
+    "com.google.guava" % "guava" % "33.4.8-jre",
+    "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonV,
+    "com.fasterxml.jackson.core" % "jackson-databind" % jacksonHotfixV,
+    "com.fasterxml.jackson.core" % "jackson-core" % jacksonV,
+    "org.yaml" % "snakeyaml" % "2.4",
+    "org.apache.commons" % "commons-compress" % "1.27.1", // workbench-libs libraries pull this in
+    "com.google.apis" % "google-api-services-pubsub" % "v1-rev20250414-2.0.0", // from workbench-google2
+    "com.google.apis" % "google-api-services-admin-directory" % "directory_v1-rev20250626-2.0.0" // from workbench-google2
   )
 
   val rootDependencies: Seq[ModuleID] = Seq(
-    "ch.qos.logback"                 % "logback-classic"     % "1.5.18",
-    "io.sentry"                      % "sentry-logback"      % "8.13.3",
-    "com.typesafe.scala-logging"    %% "scala-logging"       % "3.9.5",
-
+    "ch.qos.logback" % "logback-classic" % "1.5.18",
+    "io.sentry" % "sentry-logback" % "8.13.3",
+    "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
     "org.parboiled" % "parboiled-core" % "1.4.1",
-    "org.broadinstitute.dsde"       %% "rawls-model"         % "v0.0.449-SNAP"
-      exclude("com.typesafe.scala-logging", "scala-logging_2.13")
-      exclude("com.typesafe.akka", "akka-stream_2.13")
-      exclude("com.google.code.findbugs", "jsr305")
-      exclude("org.typelevel", "cats-parse_2.13")
-      excludeAll(excludeAkkaHttp, excludeSprayJson),
-    "org.broadinstitute.dsde.workbench" %% "workbench-util"  % s"0.10-$workbenchLibsHash",
+    "org.broadinstitute.dsde" %% "rawls-model" % "v0.0.494-SNAP"
```

Comment:
This model version bump is the only actual change in the file; everything else is IntelliJ auto-formatting